request.ts

```diff
@@ -65,9 +65,9 @@ export class PinejsClientRequest extends PinejsClientCore<PinejsClientRequest> {
 		if (params.gzip == null) {
 			params.gzip = true;
 		}
-		// We default to a 30s timeout, rather than hanging indefinitely.
+		// We default to a 59s timeout, rather than hanging indefinitely.
```

Comment:
What's special about 59secs and why does it need to be that long?

Reply:
The server will time out at the ~60s mark and having the client time out first is preferable, but having it very close means we give the server the best odds of being able to retry. Right now if it were to take 31s to respond the client would always timeout and retry when it could have worked

Reply:
Makes complete sense but my comment was a nudge really to document that thinking in the code.